requirements.txt

```diff
@@ -15,4 +15,5 @@ ctranslate2==4.5.0 # 4.4 runs cuda 12 and cudnn 8 but installing cuda via torch
 fastapi==0.115.4
 python-multipart==0.0.17 # required by our usecase with file uploads
 httpx==0.27.2 # required by starlette.testclient (FastAPI)
-uvicorn==0.32.0
\ No newline at end of file
+uvicorn==0.32.0
+transformers==4.49.0
```

Comment:
Does transformers depend on ctranslate2? If so (in either way really) verify that docker compose works with gpu passthrough. Its nice if it works for you locally but the docker compose is as close to prod as it gets.

Reply:
haven't even tested GPU at all yet xD